tests/test_bootstrap.py

```diff
@@ -98,6 +92,9 @@ def dispersy_enable_candidate_walker_responses(self):
                                                                global_time=42),
                               destination)
 
+            # The endpoint is twistified, release the reactor lock for packets to process
```

Comment:
I thought the `StandaloneEndpoint` was running on a separate thread outside the reactor?